schemas/context/action.schema.json

```
@@ -9,7 +9,7 @@
         "type": { "const": "fdc3.action" },
         "action": {
           "title": "Action Type",
-          "description": "The **action** field indicates the type of action:\n- **raiseIntent** :  If no action or `raiseIntent` is specified, then the current default behaviour will be used with `fdc3.raiseIntent` / `fdc3.raiseIntentForContext`\n- **broadcast** : If `broadcast` action is specified, it will call `fdc3.getOrCreateChannel(channelId)`to retrieve the channel, then broadcast the context to that channel with `channel.broadcast(context)`. If no, channel has been specified, it will try to broadcast to the current channel (`fdc3.getCurrentChannel()`)",
+          "description": "The **action** field indicates the type of action:\n- **raiseIntent** :  If no action or `raiseIntent` is specified, then the current default behaviour will be used with `fdc3.raiseIntent` / `fdc3.raiseIntentForContext`\n- **broadcast** : If `broadcast` action is specified, it will call `fdc3.getOrCreateChannel(channelId)`to retrieve the channel, then broadcast the context to that channel with `channel.broadcast(context)`. If no channel has been specified, it will try to broadcast to the current channel (`fdc3.getCurrentChannel()`)",
```

Apologies @symphony-jean-michael, despite having said 'all you need to do is', I just spotted another minor typo (a missing space char) - and then a couple of things that could be clearer in the description:

```suggestion
          "description": "The **action** field indicates the type of action:\n- **raiseIntent** :  If no action or `raiseIntent` is specified, then `fdc3.raiseIntent` or `fdc3.raiseIntentForContext` will be called with the specified context (and intent if given).\n- **broadcast** : If `broadcast` and a `channelId` are  specified then `fdc3.getOrCreateChannel(channelId)` is called to retrieve the channel and broadcast the context to it with `channel.broadcast(context)`. If no `channelId` is been specified, the context should be braodcast to the current channel (`fdc3.broadcast()`)",
```